Description: Disable SATA link power management by default, as it still causes disk errors and corruptions on many hardware.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/539467

--- pm/power.d/sata_alpm	2011-02-01 15:53:09.164867778 +0100
+++ pm/power.d/sata_alpm	2011-02-01 15:53:28.954867786 +0100
@@ -2,7 +2,7 @@
 
 . "${PM_FUNCTIONS}"
 
-SATA_ALPM_ENABLE=${SATA_ALPM_ENABLE:-true}
+SATA_ALPM_ENABLE=${SATA_ALPM_ENABLE:-false}
 
 help() {
 cat <<EOF
@@ -16,7 +16,7 @@
 
 This hook has 1 parameter:
 SATA_ALPM_ENABLE = whether to use SATA ALPM on battery.
-Defaults to "true".
+Defaults to "false".
 
 EOF
 }
